packages/shared/src/components/feeds/FeedContainer.tsx

```diff
@@ -118,10 +121,21 @@ export const FeedContainer = ({
   const isFinder = router.pathname === '/posts/finder';
   const isV1Search =
     searchValue === SearchExperiment.V1 && showSearch && !isFinder;
+  const shouldShowPulse =
+    checkHasCompleted(ActionType.AcceptedSearch) &&
+    !checkHasCompleted(ActionType.UsedSearch);
+
   const onSearch = (event: FormEvent, input: string) => {
     event.preventDefault();
     router.push(`${webappUrl}search?q=${encodeURIComponent(input)}`);
   };
+  const handleSearchFocus = () => {
+    if (!shouldShowPulse) {
+      return;
+    }
+
+    completeAction(ActionType.UsedSearch);
```

Comment:
Maybe we should complete this on submit rather, just heavy to use focus for it.

Reply:
Cool. It is just a bit annoying to type something while the pulse was showing, in my own experience.

Reply:
Ah good point, let's leave it :)